services/galley/src/Galley/API/Internal.hs

```diff
@@ -244,6 +247,16 @@ type IFeatureStatusPut featureName =
     :> ReqBody '[Servant.JSON] (Public.TeamFeatureStatus featureName)
     :> Put '[Servant.JSON] (Public.TeamFeatureStatus featureName)
 
+type IFeatureStatusUnlockPaymentPut featureName =
+  Summary (AppendSymbol "Unlock payment for " (Public.KnownTeamFeatureNameSymbol featureName))
+    :> "i"
+    :> "teams"
+    :> Capture "tid" TeamId
+    :> "features"
+    :> Public.KnownTeamFeatureNameSymbol featureName
+    :> "unlock"
```

Comment:
note to self: we agreed this should be a type variable `lockOrUnlock` to avoid having to duplicate this type alias.